### PR TITLE
Added contig tag to vcf in ivar_variants_to_vcf and included bcftools sort for vcf sorting

### PR DESCRIPTION
**Pull request by saramonzon**
Added reference parameter for including a contig tag in vcf file for `ivar_variants_to_vcf` script. Thanks to this tag now vcf sorting can be directly done using bcftools sort module.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/viralrecon/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/viralrecon _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by heuermh** [comment]
Curious why one check is reported here as not completed yet but the action run summary here looks ok

https://github.com/nf-core/viralrecon/actions/runs/2225674949

**Review by heuermh — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,8 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Enhancements & fixes
 
 - [[nf-core/rnaseq#764](https://github.com/nf-core/rnaseq/issues/764)] - Test fails when using GCP due to missing tools in the basic biocontainer
-- Updated pipeline template to [nf-core/tools 2.4.1](https://github.com/nf-core/tools/releases/tag/2.4.1)
-- Re-factor code of `ivar_variants_to_vcf` script.
+- Updated pipeline template to [nf-core/tools 2.3.2](https://github.com/nf-core/tools/releases/tag/2.3.2)
```
Comment (drpatelh) [inline CHANGELOG.md:11]:
```suggestion
- Updated pipeline template to [nf-core/tools 2.4.1](https://github.com/nf-core/tools/releases/tag/2.4.1)
```

File: subworkflows/local/variants_ivar.nf [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 include { IVAR_VARIANTS         } from '../../modules/nf-core/modules/ivar/variants/main'
 include { IVAR_VARIANTS_TO_VCF  } from '../../modules/local/ivar_variants_to_vcf'
-include { VCF_BGZIP_TABIX_STATS } from '../nf-core/vcf_bgzip_tabix_stats'
+include { BCFTOOLS_SORT         } from '../../modules/nf-core/modules/bcftools/sort/main'
+include { VCF_TABIX_STATS } from '../nf-core/vcf_tabix_stats'
 include { VARIANTS_QC           } from './variants_qc'
```
Comment (drpatelh) [inline subworkflows/local/variants_ivar.nf:9]:
```suggestion
include { BCFTOOLS_SORT         } from '../../modules/nf-core/modules/bcftools/sort/main'
include { VCF_TABIX_STATS       } from '../nf-core/vcf_tabix_stats'
include { VARIANTS_QC           } from './variants_qc'
```

File: subworkflows/local/variants_ivar.nf [hunk 3]
```diff
@@ -79,9 +86,9 @@ workflow VARIANTS_IVAR {
...
     multiqc_tsv     = IVAR_VARIANTS_TO_VCF.out.tsv    // channel: [ val(meta), [ tsv ] ]
 
-    vcf             = VCF_BGZIP_TABIX_STATS.out.vcf   // channel: [ val(meta), [ vcf ] ]
-    tbi             = VCF_BGZIP_TABIX_STATS.out.tbi   // channel: [ val(meta), [ tbi ] ]
-    stats           = VCF_BGZIP_TABIX_STATS.out.stats // channel: [ val(meta), [ txt ] ]
+    vcf             = BCFTOOLS_SORT.out.vcf     // channel: [ val(meta), [ vcf ] ]
+    tbi             = VCF_TABIX_STATS.out.tbi   // channel: [ val(meta), [ tbi ] ]
+    stats           = VCF_TABIX_STATS.out.stats // channel: [ val(meta), [ txt ] ]
 
     snpeff_vcf      = VARIANTS_QC.out.snpeff_vcf      // channel: [ val(meta), [ vcf.gz ] ]
```
Comment (drpatelh) [inline subworkflows/local/variants_ivar.nf:93]:
```suggestion
    vcf             = BCFTOOLS_SORT.out.vcf           // channel: [ val(meta), [ vcf ] ]
    tbi             = VCF_TABIX_STATS.out.tbi         // channel: [ val(meta), [ tbi ] ]
    stats           = VCF_TABIX_STATS.out.stats       // channel: [ val(meta), [ txt ] ]

    snpeff_vcf      = VARIANTS_QC.out.snpeff_vcf      // channel: [ val(meta), [ vcf.gz ] ]
```

**Review by drpatelh — COMMENTED** [review]
Thanks @saramonzon ! Just some aesthetic changes required.

**Comment by drpatelh** [comment]
Don't worry. I will fix these in an upcoming PR.

**Pull request merged**